packages/playground/src/types.ts

```diff
@@ -537,3 +543,68 @@ export interface FunctionSchedulesFilter {
   createdTime?: Date;
   cronExpression?: string;
 }
+
+/* ************************
+   Annotations API types
+************************ */
+
+export type AnnotatedResourceType = 'file';
+export type AnnotationType =
+  | 'diagrams.AssetLink'
+  | 'diagrams.FileLink'
+  | 'diagrams.PipeSymbol'
+  | 'documents.ExtractedText';
+export type LinkedResourceType = 'file' | 'asset';
+export type AnnotationStatus = 'suggested' | 'approved' | 'rejected';
+
+export interface AnnotationModel extends AnnotationCreate {
+  id: CogniteInternalId;
+  createdTime: Date;
+  lastUpdatedTime: Date;
+}
+
+export interface AnnotationCreate {
+  annotatedResourceType: AnnotatedResourceType;
+  annotatedResourceId?: CogniteInternalId;
+  annotatedResourceExternalId?: CogniteExternalId;
+  annotationType: AnnotationType;
+  creatingApp: string;
+  creatingAppVersion: string;
+  creatingUser: string | null;
+  data: object;
+  linkedResourceType?: LinkedResourceType;
+  linkedResourceId?: CogniteInternalId;
+  linkedResourceExternalId?: CogniteExternalId;
+  status: AnnotationStatus;
+}
+
+export interface AnnotationChangeById extends InternalId, AnnotationUpdate {}
+
+export interface AnnotationUpdate {
+  update: {
```

Comment:
`status` seems to be missing here

Reply:
Status cannot be updated, according to the service contracts, see https://docs.cognite.com/api/playground/#operation/annotationsUpdate. Does this need to be updated?

Reply:
I think it needs to be updated. How else would you approve / reject an annotation before the feedback loop is implemented? It is also allowed by the API currently, see here:
https://github.com/cognitedata/context-api/blob/master/src/api/annotations_v2/schemas.py#L161

Reply:
https://github.com/cognitedata/service-contracts/pull/1583